### DISCUSSION TITLE
Static cache with htacess redirect to /public ? (Apache)

### DISCUSSION THREAD

**Discussion by LeoSeyers** [statamic/cms#4762] [Q&A]
Hello,

Is there a way to make the static cache stragegy working with a htacess redirect to the public folder?
I mostly try to replicate envs where I only have a limited access to the host (well, just the FTP) thus the redirect. Now I'm trying to add static caching ("full" strategy) on top but I'm puzzled with how to achieve it.

Current .htacess 

``` apache
RewriteCond %{HTTP_HOST} ^.*$
RewriteCond %{REQUEST_URI} !public/
RewriteRule ^(.*)$ public/$1 [L,T=text/html]
```

From the [docs ](https://statamic.dev/static-caching#server-rewrite-rules), it appears HTML pages are internally redirected to the stache so I'm not sure this is actually doable with the htacess redirect. I did a few unsucessfull tests and was not able to access the stached HTML pages.

Otherwise here is how I'd approach it :
– Rewrite conditions to GET requests and HTML files (not sure if there's a variable to get file path without the html extension + query string)
– Rewrite rule following what is suggested in the documentation (`static/%{REQUEST_URI}_%{QUERY_STRING}\.html [L,T=text/html]`)
– Rewrite condition and rule for other assets to redirect to public

Any hints is appreciated 🍪

**Comment by LeoSeyers** [comment]
With original htacess, the stached is delivered correctly but I have the error message

`17:00:29] LOG.debug: Static cache loaded [http://atelier-choner.test/] If you are seeing this, your server rewrite rules have not been set up correctly.`

**Comment by jasonvarga** [comment]
I'm not quite sure what you're asking or doing here.

You should be able to adjust the rewrite rule we provide to point to a different directory, if that's what you're doing.
